Core/Pixel.swift

```diff
@@ -72,7 +72,6 @@ public enum PixelName: String {
     case browsingMenuWhitelist = "mb_wl"
     case browsingMenuReportBrokenSite = "mb_rb"
     case browsingMenuSettings = "mb_st"
-    case browsingMenuFindInPage = "mb_fp"
```

Comment:
I think this change was unintentional (prolly caused by merge).

Reply:
sorry about that.. it was accidental while removing my pixel tag..